src/Domain/CoordinateMaps/CoordinateMap.tpp

```diff
@@ -374,6 +374,18 @@ CoordinateMap<SourceFrame, TargetFrame, Maps..., NewMap> push_back_impl(
       std::move(std::get<Is>(old_map.maps_))..., std::move(new_map)};
 }
 
+template <typename... NewMaps, typename SourceFrame, typename TargetFrame,
+          typename... Maps, size_t... Is, size_t... Js>
+CoordinateMap<SourceFrame, TargetFrame, Maps..., NewMaps...> push_back_impl(
+    CoordinateMap<SourceFrame, TargetFrame, Maps...>&& old_map,
+    CoordinateMap<SourceFrame, TargetFrame, NewMaps...> new_map,
```

Comment:
why is the `old_map` taken by rvalue reference and the `new_map` by value? they seem to be treated the same

Reply:
For consistency with the other variants :man_shrugging: